db/migrate/20150911195321_add_budget_roles.rb

```diff
@@ -0,0 +1,24 @@
+class AddBudgetRoles < ActiveRecord::Migration
+  def change
+    User.with_email_role_slug!(
```

Comment:
does this comment apply to the db/seeds.rb file as well? The method is also used there, and run on every rspec.

How best to DRY with code/logic used in migrations and elsewhere?